weco-deploy/README.md

```diff
@@ -0,0 +1,14 @@
+# wellcome/publish_service
```

Comment:
```suggestion
# wellcome/weco-deploy, surely?
```

Reply:
sorry, thought I removed those changes - this change shouldn't include weco-deploy

Reply:
the container is built in the weco-deploy repo, not here